code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/reactive_armour_shell
```

Comment:
Might want to fix the typepaths while you're at it. Just make sure to have like, an UpdatePaths script for it, because it might appear on some maps.

Reply:
only 4 instances :)